build/Linux+BSD/org.musescore.MuseScore.appdata.xml.in

```diff
@@ -91,19 +90,10 @@
       <image>https://raw.githubusercontent.com/flathub/org.musescore.MuseScore/master/screenshots/07-new-score-wizard.png</image>
     </screenshot>
   </screenshots>
-
-  <developer_name>MuseScore BVBA</developer_name>
-  <update_contact>jurf_AT_riseup.net</update_contact>
-  <translation>qt</translation>
-  
-  <kudos>
-    <kudo>ModernToolkit</kudo>
-    <kudo>UserDocs</kudo>
-  </kudos>
-
+  <content_rating type="oars-1.1"/>
```

Comment:
The empty tag produces a warning in `appstreamcli validate`.

Reply:
Oh, well, it’s a valid entry though.

Actually, I tried doing the OARS generation again and it’s no longer empty, so I’ll add it there.